src/nvim/version.c

```diff
@@ -2709,7 +2709,7 @@ void intro_message(int colon)
   static char *(lines[]) = {
     N_(NVIM_VERSION_LONG),
     "",
-    N_("by Bram Moolenaar et al."),
+    N_("by al."),
```

Comment:
My first impulse was to suggest to remove the message completely, as saying "made by someone" resp. "Neovim, made by the Neovim people" seems quite redundant.

In general, I like the idea of keeping it as a reference, but I only "got it" after reading your comment (and seeing the diff of old/new message on Github), so I'm not sure.